docs/pages/modules/module-api.mdx

```diff
@@ -152,6 +152,44 @@ async function getMessageAsync() {
 }
 ```
 
+</APIBox>
+<APIBox header="SuspendFunction" platforms={["android"]}>
```

Comment:
It's still `AsyncFunction`, so `SuspendFunction` is not a separate component, right? If so, I think it would be better to move these notes to `AsyncFunction` section, as an alternative version for async functions. Then we also don't have to copy the example 😅